internal/test/windows/windows.go

```diff
@@ -142,7 +142,7 @@ func (w *Windows) Run(cmd string, psCmd bool) (string, error) {
 
 	out, err := session.CombinedOutput(cmd)
 	if err != nil {
-		return "", err
+		return string(out), err
 	}
 	return string(out), nil
```

Comment:
I feel the `if` statement can be removed avoiding the condition evaluation,  and the   `return string(out), err`  may work just fine.

cc: @aravindhp

Reply:
Good catch. I have made the change. We should also make this change on the WMCO side. @jrvaldes this might a good first PR for you.

Reply:
Looking forward to it, just shed the lights